editor/components/inserter/index.js

```diff
@@ -54,34 +101,58 @@ class Inserter extends Component {
 		}
 
 		return (
-			<Dropdown
-				className="editor-inserter"
-				position={ position }
-				onToggle={ this.onToggle }
-				expandOnMobile
-				headerTitle={ title }
-				renderToggle={ ( { onToggle, isOpen } ) => (
-					<IconButton
-						icon="insert"
-						label={ __( 'Add block' ) }
-						onClick={ onToggle }
-						className="editor-inserter__toggle"
-						aria-haspopup="true"
-						aria-expanded={ isOpen }
-					>
-						{ children }
-					</IconButton>
-				) }
-				renderContent={ ( { onClose } ) => {
-					const onSelect = ( item ) => {
-						onInsertBlock( item );
-
-						onClose();
-					};
-
-					return <InserterMenu onSelect={ onSelect } />;
-				} }
-			/>
+			<div>
```

Comment:
Is this `div` necessary? Would it work with `Fragment` component?

Reply:
The `div` is not necessary, so I changed it to a `Fragment` component as you suggested. Thanks for noticing that!